internal/errcheck/errcheck.go

```diff
@@ -220,12 +219,31 @@ func (v *visitor) ignoreCall(call *ast.CallExpr) bool {
 			if re, ok := v.ignore[pkg.Path()]; ok {
 				return re.MatchString(id.Name)
 			}
+
+			// if current package being considered is vendored, check to see if it should be ignored based
+			// on the unvendored path.
+			if nonVendoredPkg, ok := nonVendoredPkgPath(pkg.Path()); ok {
+				if re, ok := v.ignore[nonVendoredPkg]; ok {
```

Comment:
Adding this as a second check so that any existing `ignore` entries that use fully qualified vendor paths will still work just as they did before. This extra check basically just "unvendors" the package being considered and uses the ignore entry there.

I think this is the correct behavior and reflects the intent of #102.